scripts/cromwell/analyze_resource_acquisition.py

```diff
@@ -90,11 +92,12 @@ def calculate_start_end(call_info, override_warning=False, alias=None):
   """
   Modified from: https://github.com/broadinstitute/dsde-pipelines/blob/develop/scripts/calculate_cost.py
   """
-  job_id = call_info['jobId'].split('/')[-1]
-  if alias is None or alias == "":
-    alias = job_id
-  else:
-    alias += "." + job_id
+  if 'jobId' in call_info:
+    job_id = call_info['jobId'].split('/')[-1]
+    if alias is None or alias == "":
+      alias = job_id
+    else:
+      alias += "." + job_id
```

Comment:
So if `jobId` is absent, `alias` may end up at `None`, although it seems that `alias` is only used for a warning message in this script. Perhaps add an `else` to the first `if` to assign `alias` explicitly to something like `NA`. It may also be helpful to emit a warning explaining what situations can cause this to happen.

Reply:
I don't think I've ever encountered a scenario where alias is None at this point (and I don't think it's really possible within the context of the larger script), which is why I didn't bother with that condition, but I've added a catch just in case

Reply:
The new change leaves no path for alias to be None, so I think it solves the issue.